harfbuzz.lua

```diff
@@ -19,45 +19,57 @@ includedirs {
 
 files {
     "./src/hb-aat-layout.cc",
+    "./src/hb-aat-map.cc",
+	"./src/hb-blob.cc",
```

Comment:
nitpick: Spaces instead of tabs

Reply:
Good spot @chri7325 . Should be fixed now